friends/src/FriendsList.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+export default class FriendsList extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: []
+    };
```

Comment:
I would hold your `friends` state and `CDM` in the `App.js` component. Reason being is now only children of `FriendsList` can have access to the `friends` state and methods where as `App.js` is the most root component besides `Index.js`, meaning is can pass it's state and methods to any component that needs it.